comp/core/sysprobeconfig/sysprobeconfigimpl/config_mock.go

```diff
@@ -51,6 +64,12 @@ func newMock(deps dependencies, t testing.TB) sysprobeconfig.Component {
 		}
 	})
 
+	// Overrides are explicit and will take precedence over any other
+	// setting
+	for k, v := range deps.Params.Overrides {
+		config.SystemProbe.SetWithoutSource(k, v)
+	}
```

Comment:
What do you mean by `will take precedence over any other setting` ?
I think any setting set with a higher priority than `Unknown` will still be used ?

Considering it's in a test I don't think it can happen unless you explicitly set it up in the test, but the commit is still unclear

Reply:
When calling `SetWithoutSource` the value overwrites everything else.

Reply:
I just checked the source and `SetWithoutSource` just calls `Set(..., SourceUnknown)`
`SourceUnknown` is the second least important priority (behind default), so any of file, env var, runtime, RC, or CLI, would take precedence
Wouldn't it ?